cookbook/category/add-properties-to-category.rst

```diff
@@ -40,7 +44,7 @@ You also need to configure the mapping override in your application configuratio
                 override: Acme\Bundle\CatalogBundle\Entity\Category
 
 Define the Category Class
--------------------------
+*************************
 
 You need to update your category entity parameter used in ``entities.yml`` file (if you are creating a new bundle, double check that this file is inside the extension):
```

Comment:
May be wee can show it like

```php
// AcmeBundleExtension.php

// code
        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
        $loader->load('entities.yml');

// more code
```

or make it bold

**double check that file ```entities.yml``` is inside the extension**